src/OrbitQt/orbitmainwindow.h

```diff
@@ -172,6 +172,7 @@ class OrbitMainWindow : public QMainWindow {
   std::unique_ptr<OrbitGLWidget> introspection_widget_ = nullptr;
   QFrame* hint_frame_ = nullptr;
   QLabel* target_label_ = nullptr;
+  std::string orbit_executable_path_;
```

Comment:
I think there's no need to have this as a member, is there? It's used only in one place (`OpenCapture`) and can be determined easily there instead of in the constructor, so this feels a bit like unneeded bloat for the class.

Reply:
Changed accordingly. Thanks for point out this!

I was thinking this is computed every time so I added it as a member. But I was also considering that it would be a little bit redundant if added as a member. Thanks for helping me to make the decision!